### ISSUE TITLE
RC Test: John Travoltage 1.6.0-rc.3

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/qa#697]
@emily-phet , @terracoda , @KatieWoe john-travoltage/1.6.0-rc.3 is ready for RC testing. Please document issues in https://github.com/phetsims/john-travoltage/issues and link to this issue.

This release includes PhET brand with accessibility features. It does not inclue PhET-iO.

This is an RC spot check following https://github.com/phetsims/qa/issues/693. Two issues were found that should be verified in this build:

- [x] [Update QA Credits](https://github.com/phetsims/john-travoltage/issues/446)
- [x] [Verify fix for Android App](https://github.com/phetsims/john-travoltage/issues/443)

Then please do standard spot checking for the RC on a platform or two. No need to test any screen reader functionality again.

**Link to Sim**:  https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-rc.3/phet/john-travoltage_en_phet.html

- [x] Tester 1 (Tester = KW, Platform = Chromebook, Time = .5hrs {max 2 hours}
- [x] Tester 2 (Tester = NS , Platform =  Mac + safari , Time = 2 hrs {max 2 hours}

Assigning to @KatieWoe and @kathy-phet for prioritization.

**Comment by KatieWoe** [comment]
QA is done

**Comment by jessegreenberg** [comment]
Thank you QA team! 1.6 is Ready for publication, closing this issue.
